### PR TITLE
feat: Update AWS Services (new fields)

### PR DESCRIPTION
**Pull request by bbernays**
#### Summary

This PR was created by running:
```
go list -m all | grep github.com/aws/aws-sdk-go-v2 | awk '{print $1}' | xargs go get -u && go mod tidy && go run codegen/main.go && make gen-docs
```

### CODE REVIEW

File: .github/workflows/regenerate_aws_resources.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Regenerate AWS Resources after upgrading packages
```
Comment (bbernays) [inline .github/workflows/regenerate_aws_resources.yml:1]: @erezrokah - How does this look?

File: .github/workflows/regenerate_aws_resources.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+on:
+  schedule:
+    - cron: "30 * * * *"
```
Comment (bbernays) [inline .github/workflows/regenerate_aws_resources.yml:4]: We should do this at a different time than the endpoints.

File: .github/workflows/regenerate_aws_resources.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        uses: peter-evans/create-pull-request@v4
+        with:
+          # required so the PR triggers workflow runs
+          token: ${{ secrets.GH_CQ_BOT }}
+          branch: fix/update_aws_packages
+          base: main
+          title: "fix(deps): Update AWS packages"
+          commit-message: "fix(deps): Update AWS packages"
+          body: This PR was created by a scheduled workflow to upgrade and regenerate resources
+          author: cq-bot <cq-bot@users.noreply.github.com>
```
Comment (bbernays) [inline .github/workflows/regenerate_aws_resources.yml:37]: Removed the automerge label to protect against potential breaking changes if fields change dramatically

**Comment by yevgenypats** [comment]
Love the `bash` line!

**Review by hermanschaaf — APPROVED** [review]
Really neat that this could be done fully automatically now :)

**Pull request merged**